components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/java/org/wso2/carbon/identity/api/server/idp/v1/impl/FederatedAuthenticatorConfigBuilderFactory.java

```diff
@@ -238,7 +238,7 @@ private static void validateAuthenticatorProperties(String authenticatorName, Li
 
         if (!areAllDistinct(properties)) {
             Constants.ErrorMessage error = Constants.ErrorMessage.ERROR_CODE_INVALID_INPUT;
-            throw new IdentityProviderManagementClientException(error.getCode(), error.getDescription());
+            throw new IdentityProviderManagementClientException(error.getCode(), error.getMessage());
```

Comment:
What about the description was that returned before ?

Reply:
Before when buidling APIError, the error message used for the both error message and description. https://github.com/wso2/identity-api-server/blob/78ad1f71ec10c9cca45eff9def75b2f0fb928f6a/components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/java/org/wso2/carbon/identity/api/server/idp/v1/core/ServerIdpManagementService.java#L3318

Reply:
If so this test case is wrong https://github.com/wso2/product-is/blame/master/modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/idp/v1/IdPFailureTest.java#L208-L211

@Shenali-SJ